### PR TITLE
add 'direction' option

### PR DESCRIPTION
**Pull request by jackyon**
add 'direction' option for the scrollTop/scrollLeft method

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ import scrollToWithAnimation from 'scrollto-with-animation'
 ```javascript
 scrollToWithAnimation(
     document.body, // element to scroll
+    'scrollTop' // direction to scroll
```
Comment (davesnx) [inline README.md:40]: I prefer to have a option that is direction: `vertical` or `horitzontal`

**Comment by davesnx** [comment]
Thanks for add this direction method, in case that there so much options, we should thing about create and object for passing all of them.

**Comment by kof** [comment]
@davesnx any updates? I also need this option.

**Comment by davesnx** [comment]
@kof Will be on the next version

**Pull request merged**